### ISSUE TITLE
learning rate  

### ISSUE THREAD

**Issue by Isabelliu** [experiencor/keras-yolo3#109]
Hi, thank you for your excellent work.
Can we chang the learning rate while training? 
eg. the first 50 epochs 0.001, the last 50 epochs 0.0001?

**Comment by PhilippMarquardt** [comment]
Just write your own callback.
